### PR TITLE
Fix margins and insets on Home scene

### PR DESCRIPTION
**Pull request by samholmes**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [ ] Yes
- [x] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206423292872435

### CODE REVIEW

**Comment by Jon-edge** [comment]
Broken on Android

**Review by Jon-edge — CHANGES_REQUESTED** [review]
Wallet list margins didn't work for both platforms. Also I updated the task to include tx list scene since it also used the undoInsets

**Review by Jon-edge — APPROVED** [review]

File: src/components/themed/TransactionListRow.tsx [hunk 1]
```diff
@@ -189,7 +189,7 @@ export function TransactionListRow(props: Props) {
   })
 
   return (
-    <CardUi4 icon={icon} onPress={handlePress} onLongPress={handleLongPress}>
+    <CardUi4 icon={icon} onPress={handlePress} onLongPress={handleLongPress} marginRem={[0.5, 1]}>
```
Comment (samholmes) [inline src/components/themed/TransactionListRow.tsx:193]: I thought we wanted .5 rem around all components and then a 0.5 padding around the outer-most container component on a scene?

File: src/components/themed/WalletListHeader.tsx [hunk 2]
```diff
@@ -48,12 +47,6 @@ export class WalletListHeaderComponent extends React.PureComponent<Props> {
             <SectionHeaderUi4 leftTitle={lstrings.title_wallets} rightNode={addSortButtons} />
           </EdgeAnim>
         )}
-
-        {searching ? null : (
-          <EdgeAnim enter={{ type: 'fadeInUp', distance: 20 }} exit={{ type: 'fadeOutUp' }}>
-            <PromoCard navigation={navigation} />
-          </EdgeAnim>
-        )}
```
Comment (samholmes) [inline src/components/themed/WalletListHeader.tsx:56]: This isn't in the scope of this PR, so could you give me the task to understand what/why for this change?

File: src/components/themed/TransactionListRow.tsx [hunk 3]
```diff
@@ -188,13 +188,14 @@ export function TransactionListRow(props: Props) {
...
 
+  // HACK: Handle 100% of the margins because of SceneHeader usage on this scene
   return (
     <CardUi4 icon={icon} onPress={handlePress} onLongPress={handleLongPress} marginRem={[0.5, 1]}>
       <SectionView dividerMarginRem={[0.2, 0.5]} marginRem={[0.25, 0]}>
         <>
           <View style={styles.row}>
             <EdgeText style={styles.titleText}>{name}</EdgeText>
-            <EdgeText style={styles.titleText}>{cryptoAmountString}</EdgeText>
+            <EdgeText style={styles.cryptoText}>{cryptoAmountString}</EdgeText>
```
Comment (samholmes) [inline src/components/themed/TransactionListRow.tsx:198]: I don't know what this change is either. Provide task and screenshot

File: src/components/themed/WalletListHeader.tsx (hunk 2 above)
Comment (samholmes) [reply src/components/themed/WalletListHeader.tsx:56]: Also, provide screenshot if applicable

File: src/components/themed/TransactionListRow.tsx (hunk 1 above)
Comment (Jon-edge) [reply src/components/themed/TransactionListRow.tsx:193]: Yes, ideally, but we're hacking around the fact that SceneHeader was never updated to follow these rules.

Comment (samholmes) [reply src/components/themed/TransactionListRow.tsx:193]: performance.measure('myMeasure', 'myMark')

Comment (samholmes) [reply src/components/themed/TransactionListRow.tsx:193]: Can you clarify what you mean when you refer to the SceneHeader component?

File: src/components/themed/TransactionListRow.tsx (hunk 3 above)
Comment (samholmes) [reply src/components/themed/TransactionListRow.tsx:198]: image (1)

**Pull request merged**